airflow/providers/amazon/aws/transfers/s3_to_sql.py

```diff
@@ -120,7 +124,7 @@ def execute(self, context: Context) -> None:
     @cached_property
     def db_hook(self):
         self.log.debug("Get connection for %s", self.sql_conn_id)
-        hook = BaseHook.get_hook(self.sql_conn_id)
+        hook = BaseHook.get_hook(self.sql_conn_id, hook_params=self.sql_hook_params)
```

Comment:
You can’t do this because providers needs to support older Airflow versions.

Reply:
You are right, I am going to switch this to a draft and look into a different approach